src/constants.js

```diff
@@ -12,7 +12,6 @@ export const VSMShadowMap = 3;
 export const FrontSide = 0;
 export const BackSide = 1;
 export const DoubleSide = 2;
-export const TwoPassDoubleSide = 3;
```

Comment:
`TwoPassDoubleSide` was included in r148 so we need a deprecation warning.

Reply:
I don't think we need to do that when a feature didn't last more than a single release.

Reply:
I think quite a many users have upgraded. It would not be good to just break their code without a warning... We can retain the constant and warn somewhere (e.g. when `material.side` is set) if it is used, similar how it was done with `RGBFormat`.

Reply:
I'll restore the constant for now.